scripts/build.ps1

```diff
@@ -55,6 +55,7 @@ Param(
 )
 
 $ErrorActionPreference = 'Stop'
+$ErrorView = 'Normal'
```

Comment:
This disables the summary view in Pwsh that tries to summarize the error. We want to see the error in full.